app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -46,14 +46,27 @@ def service_provider_homepage_url
         sp_return_url_resolver.homepage_url if service_provider
       end
 
-      def usps_outage_message_enabled
-        IdentityConfig.store.in_person_usps_outage_message_enabled
+      def outage_message_enabled
+        IdentityConfig.store.in_person_outage_message_enabled
+      end
+
+      def date
+        format_date(IdentityConfig.store.in_person_outage_expected_update_date)
+      end
+
+      def email_date
+        format_date(IdentityConfig.store.in_person_outage_emailed_by_date)
       end
 
       private
 
       attr_reader :enrollment
 
+      def format_date(date)
+        d = date.to_date
+        d.strftime(I18n.t('date.formats.short'))
```

Comment:
@gina-yamada thank you for finding the bug! I tracked it down to the formatting - when i used %A for some reason the day of the week was off by one. I separated out the conversion of the string to date from the formatting and that seems to be working now.

Reply:
this change fixed the day of the week name issue but left the month name and day of week name untranslated